sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -43,6 +44,7 @@ class ProjectOverviewView extends VerticalLayout {
 
     private Grid<ProjectSummary> projectGrid
     private HorizontalSplitPanel splitPanel
+    private final Collection<String> columnIdsWithFilters = ["ProjectCode", "ProjectTitle"]
```

Comment:
I think this can even be static 😄 nice idea putting this here at the top

Reply:
Done